integrationTests/multiShard/smartContract/scCallingSC_test.go

```diff
@@ -1053,6 +1167,35 @@ func putDeploySCToDataPool(
 	return scAddressBytes
 }
 
+func putNormalTxToDataPool(
```

Comment:
integrationTests.CreateAndSendTransaction instead of this function.

Reply:
Replaced.